### DISCUSSION TITLE
Bump MacKernelSDK

### DISCUSSION THREAD

**Discussion by knxoff** [ChefKissInc/NootedRed#281] [General]
Can't downloading Bump MacKernelSDK #1115

**Comment by knxoff** [comment]
i need the "artifacts"

**Reply by AlfCraft07** [reply]
Download from nightly release

**Reply by knxoff** [reply]
and where do I find that?

**Reply by AlfCraft07** [reply]
Under releases. It was actually made to simplify the download process.

**Reply by VisualEhrmanntraut** [reply]
https://github.com/ChefKissInc/NootedRed/releases

**Reply by nononymousse** [reply]
for future people reading this, to see the correct date you need to open "Assets" in the releases page and then you can see the date of the files (i thought it wasn’t updated in months until i saw that)
